### ISSUE TITLE
[Bug Report][3.5.11] [VTooltip] 

### ISSUE THREAD

**Issue by Doltario** [vuetifyjs/vuetify#19512]
### Environment
**Vuetify Version:** 3.5.11
**Vue Version:** 3.4.8
**Browsers:** Mozilla Firefox
**OS:** Mac OSX

### Steps to reproduce
Nothing to do but to observe that the tooltip is not vertically nudged.

### Expected Behavior
From that [comment](https://github.com/vuetifyjs/vuetify/issues/14451#issuecomment-1171058389) I expected the second part of the `offset` prop to, in that case (tooltip positionned at the end of the activator) to nudge the tooltip vertically. Also, I didn't find any other documentation on that, so I may be doing something wrong.

### Actual Behavior
The first `main` part of `offset="main cross"` works perfectly, but the `cross` does not seem to have any effect.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqdks9qAyEQxl9l8Fy7ueRStoEW+gTNrfZg1AWL64iOklL67p3uZkMOOSR7kfnjb76PYT5+RMmme0npsVUnnkRPbkxBk9upCNBb38AEXcqzElYOwR3hqxbyw7csSRsndcYaLYwkt0pMDFNNHiieEoB30pmWhHuEGMgn0IZ804SZZyedXSQlIKDR5DFyrUyc2O3n/313RhcdrsxCV3Xfol2h6pgSMKJ1QTYdqgMchuKIWxvYbtJRiWUqwBVvS/M2j3s8A3d4JKbW7+UViXBcIXuYwZuU/2M+Ho767uKkxO9DrCFMz+cfcQbQCg==)

**Comment by elkofy** [comment]
Hi, don't be harsh with me this my first time. I have been looking at the issue and it is mentionned that the cross section is used along the element . Since it is positioned to the right maybe that's what makes it impossible ?

**Comment by johnleider** [comment]
You're not providing a valid value. It's string | number | number[] https://vuetifyjs.com/en/api/v-menu/#props-offset

https://play.vuetifyjs.com/#eNqdkr9OxDAMxl/Fykwoyy2onAQST8BtlCGXuFJQGkeJE4EQ745p6Ynhhrsukf/k5++T5dcvVbLtHlO6bRXVveoZpxQM436IAL3zDWwwpTwMyukx4Ae818J+/NQlGYvaZKrRwcR6N6iZEarpI8e/BOCFTeY1kR4TBfYJjGXfDFOW2clkjDwoCGQNe4pSKzOn9oflf9+d0FVHKovQWd3n6DaoolAKJnIYdDOhItA4FmRp7e4GtU4EOONrbV7m70An4Ap/LNT2nTwRM00bZI8LeJHybyyHI1Hf/Tsn9X0Tawjz8/YDSt/O0g==

If you have any questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Comment by Doltario** [comment]
Hi, 

I apologise if I didn’t provide a clear explanation of the problematic.

In that case, when the tooltip is at the end (on the right) of the activator, I am trying to change its **vertical** offset. I genuinely think this props doesn’t work as Kael  described it in the comment I attached in my first message. 

> nudge has also been replaced with offset="main cross" where main is a distance away from the activator side and cross is along it.

But the `cross` doesn’t seem to do anything. I tried with `0 50px` to observe that the `0` horizontal offset actually works, but not the vertical `50px`. I agree that `50px` might not have been a good example value, but I tried with `offset=“0 50”` as well and it kept behaving the same.

Here is a more complete [example](https://play.vuetifyjs.com/#eNrFlsFu4jAQhl9llF5JnbJwibJIHPa0UnvYij2Uamucgbjr2JbtZKlWffc6KRCXUiqkoF6iZH7b8/2eiZO7/5E1jEy1vqwrjNIoc1hqQR1O5hIgy3kNTFBrv88jTePRPGrjXimGkxkaxxkVoJZLiw6UhNmtUsJxnRGvb0aGa+TxUuAaHivr+PIptpoyjKlRlcyhdPEYynWQxE+u44WTu0eAHzK/uOoeve5eMwJljtfUKdOiGpRuHoFQjDqupI+hzH2gVDmKuKaiwg22l5J51C0JMPnlDJcreEgeMrJL0I3o6Lwa4h2kHfZEm36Ie12VCzSQnAs2THaIO9T3LYRa4CYMd8bukgFcJcn9nr+pMfQJtuIpLjPim+9sfTjqvQ8bgx/1YqP10Y/jLyjxW4d6fcyjXp/kclviA0X2hQxrOGVMmbxJ5BSku9Vol6IwuPTzCue0TQnxJ2LTHY/2kqmSoCRUc5+7RFmRC22UtvGrsdCOo2bVWv2zEFT+DaXJyQtnZEcXFlVw64KqbiIx92d3EG4Fzvyp3Fya/ch5zApkHgqYEm2BbcUYWuu3qdnWZuCbFdrjNoUZFTyHtqwDuPkZpPaTDuU+M9Lw3EhCWQyQ0BhljgF92wNaYEFrrgxYxNJCJVlB5QrzAVzf3PaOW6DQAe0/aqRv82O8oxR+G+VfhU95qX33zf0S4nHvxNvou2/F9i4jwc9Q9DyQlRDt5f4FcvG3Lw==) with different values I tried.

Did the comment of Kael become untrue ? Anyway, the documentation might be lacking some information, about what the prop should look like as a string.

Thanks to anyone taking time reading.
